cdk.json

```diff
@@ -15,6 +15,16 @@
     ]
   },
   "context": {
+    "deploy-where": "dev",
+    "dev": {
+      "sds_id": "dev",
```

Comment:
Is there a way to use a different sds_id besides updating this file? For example, we used to set the SDSID to something unique as part of the deploy command like SDSID=hoyt-testing so our resources wouldn't conflict.

Reply:
I think you could change this to what you need in your file so it is distinct from others. Although if you wanted to deploy the stacks twice then you'd need to have two versions of this file I think...